ompi/mca/osc/ucx/osc_ucx_comm.c

```diff
@@ -763,17 +1009,29 @@ int ompi_osc_ucx_get_accumulate(const void *origin_addr, int origin_count,
             if (ret != OMPI_SUCCESS) {
                 return ret;
             }
-
```

Comment:
Can we remove this?

Reply:
What is 'this'? The whitespace? It's part of a longer section of code that was removed.

Reply:
ok